c-pallets/sminer/src/lib.rs

```diff
@@ -1160,7 +1160,7 @@ impl<T: Config> Pallet<T> {
 			T::AScheduler::schedule(
 				DispatchTime::At(buffer_period),
 				None,
-				59,
+				59, // ?? const
```

Comment:
Here is a normal transaction without special requirements, so I use `LOWER_ PRIORITY`:
https://github.com/CESSProject/cess/pull/88/files#diff-30c617cf01d014c0e882cf4c64b66e5f459de77270aa8617a71c6fc93a9a143aR1166